contribs/gmf/less/mobile.less

```diff
@@ -130,6 +130,55 @@ span.twitter-typeahead {
   }
 }
 
+#measure-tools {
+  .measure-tools-button {
+    display: block;
+    margin-bottom: 5px;
+  }
+}
+
+div[gmf-mobile-measure-length],
+div[gmf-mobile-measure-point] {
```

Comment:
What about using a className for that? (for example: `.measure-tools-control`)
I'm not sure. What's your opinion?